package.json

```diff
@@ -39,5 +39,8 @@
     "action-version": "npm version prerelease --preid=\"${PR_BRANCH_NAME}\" --no-git-tag-version",
     "test": "gscan .",
     "test:ci": "gscan --fatal --verbose ."
+  },
+  "dependencies": {
+    "sqlite3": "^5.0.0"
```

Comment:
My ghost install was showing errors. npm install sqlite3 fixed the error for me

Reply:
Maybe it doesn’t matter that much because I’m not sure how this is used in an actual theme build, but I think this kind of thing would usually be in `devDependencies` instead of `dependencies` because it’s not needed for production 🤓